File: bottle.go
package main

import (
    "fmt"
    "time"
)

func main() {
    nbeer := 99
    world := "bottles"
    for nbeer > 0 {
        if (nbeer == 1) {
            world = "bottle"
        }
        fmt.Println(nbeer, " ", world, " of beer on the wall. Take one down")
        nbeer -= 1
        if (nbeer > 0) {
            fmt.Println(nbeer, " ", world, " of beer on the wall")
        } else {
            time.Sleep(time.Second * 20)
            fmt.Println("No more beer on the wall")
        }
    }
}

File: bottle.py

from time import sleep


def main():
    numBeer = 99
    world = "bottles"
    while numBeer > 0:
        if numBeer == 1:
            world = "bottle"
        print(str(numBeer) + " " + world + "of beer on the wall. Take one down.")
        numBeer = numBeer - 1
        if numBeer > 0:
            print(str(numBeer) + " " + world + " of beer on the wall")
        else:
           sleep(20)
           print("No more bottles of beer on the wall")

main()
